content/en/blog/_posts/2024-05-13-Gateway-API-v1.1/index.md

```diff
@@ -0,0 +1,199 @@
+[list of Gateway API implementations](https://gateway-api.sigs.k8s.io/implementations/#service-mesh-implementation-status).
+As an example, one could do a canary deployment of a workload deep in an application's call graph with an HTTPRoute as follows:
+
+~~~
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: color-canary
+  namespace: faces
+spec:
+  parentRefs:
+    - name: color
+      kind: Service
+      group: core
+      port: 80
+  rules:
+  - backendRefs:
+    - name: color
+      port: 80
+      weight: 50
+    - name: color2
+      port: 80
+      weight: 50
+~~~
+      
+This would split traffic sent to the `color` Service in the `faces` namespace 50/50 between the original `color` Service and the `color2` Service, using a portable configuration that's easy to move from one mesh to another.
+
+Leading Contributors: Flynn, John Howard, Keith Mattix, and Mike Morris
+
+#### [GRPCRoute](https://gateway-api.sigs.k8s.io/guides/grpc-routing/)
+If you are already using the experimental version of GRPCRoute, we recommend holding
+off on upgrading to the standard channel version of GRPCRoute until the
+controllers you're using have been updated to support GRPCRoute v1. Until then,
+it is safe to upgrade to the experimental channel version of GRPCRoute in v1.1
+that includes both v1alpha2 and v1 API versions.
+
+Leading Contributor: Richard Belleville
+
+#### [ParentRef Port](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io%2fv1.ParentReference)
+Now you can use the `port` field in ParentRefs to attach resources to Gateways,
+Services, or other parent resources. For example, you can attach an HTTPRoute to
+one or more specific Listeners of a Gateway based
+on the Listener `port`, instead of `name` field.
+
+Leading Contributor: Frank Budinsky
+
+#### [Conformance Profiles and Reports](https://gateway-api.sigs.k8s.io/concepts/conformance/#conformance-profiles)
+
+The Conformance report API has been expanded with the `mode` field (intended to
+specify the working mode of the implementation), and the `gatewayAPIChannel`
+(standard or experimental). The `gatewayAPIVersion` and `gatewayAPIChannel` are
+now filled in automatically by the suite machinery, along with a brief
+description of the testing outcome. The Reports have been reorganized in a more
+structured way, and the implementations can now add information on how the tests
+have been run and provide reproduction steps.
+
+Leading Contributors: Christine Kim, Mattia Lavacca, Shane Utt
+
+### New Additions to Experimental Channel
+
+#### [Gateway Client Certificate Verification](https://gateway-api.sigs.k8s.io/geps/gep-91/)
+
+Gateways can now configure client cert verification for each Gateway Listener by
+introducing a new field `frontendValidation` field within `tls`. This field
+supports configuring a list of CA Certificates that can be used as a trust
+anchor to validate the certificates presented by the client.
+
```

Comment:
```suggestion

This example configuration shared below, shows how the CA Certificate stored in the `foo-example-com-ca-cert` ConfigMap can be used to validate the certificates presented by clients connecting to the `foo-https` Gateway listener.

```
apiVersion: gateway.networking.k8s.io/v1
kind: Gateway
metadata:
  name: client-validation-basic
spec:
  gatewayClassName: acme-lb
  listeners:
  - name: foo-https
    protocol: HTTPS
    port: 443
    hostname: foo.example.com
    tls:
      certificateRefs:
      - kind: Secret
        group: ""
        name: foo-example-com-cert
      frontendValidation:
        caCertificateRefs:
        - kind: ConfigMap
          group: ""
          name: foo-example-com-ca-cert
```
```

Reply:
@arkodg I tried to fix this up.  Let me know if there are any mistakes.